server.go

```diff
@@ -115,6 +121,25 @@ func (s *server) initNoteServiceClient() {
 	s.noteService = noteService
 }
 
+func (s *server) InitEnv() {
+	err := godotenv.Load()
+	if err != nil {
+		fmt.Println("Error loading .env file")
+		return
+	}
+	var GOOGLE_APP_ID string = os.Getenv("GOOGLE_APP_ID")
+	var GOOGLE_APP_SECRET string = os.Getenv("GOOGLE_APP_SECRET")
```

Comment:
we don't use go.Getenv in production but env flag such as --port, --env ..., 
there are exemple in main.go, you can look at gmail-super-secret for the flow,
Let me know if you have any question

Reply:
I changed the management of the secrets, I pass in parameter as you make it on the gmail-super-secret, I don't know if it will work on production environnement ?